src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,36 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  static alive = [];
+
+  hidden = false;
```

Comment:
[FUNCTION NAMING] The `hidden` property should only exist in the `Herbivore` class, as it is not relevant to the `Carnivore` class or the `Animal` class. Please move the `hidden` property to the `Herbivore` class.